core/src/config/tasks.ts

```diff
@@ -4,7 +4,7 @@ export const DEFAULT = {
       _toExpand: false,
 
       // Should this node run a scheduler to promote delayed tasks?
-      scheduler: process.env.SCHEDULER === "true",
+      scheduler: true,
```

Comment:
Maybe a better approach would be to have it default to `true`, but provide the option to set it to false? And then we could still leave it out of the basic examples and only include it where we want to disable it?

Reply:
Or could we use `WEB_SERVER` to determine this value?

Reply:
I'd suggest using `process.env.WORKERS` (ie https://github.com/grouparoo/grouparoo/blob/master/core/src/config/tasks.ts#L51-L53 ) to set it.  If you have > 0 workers, turn on the scheduler too.

You generally don't want the web servers working on background tasks.